### ISSUE TITLE
nanopaint tiles sprite issues

### ISSUE THREAD

**Issue by Eneocho** [d3athrow/vgstation13#37322]
![image](https://github.com/user-attachments/assets/be11e40b-98f5-477a-8ad6-98f831d8b813)
the sprites of stuff over nanopaint turn semitransparent and get blended with the color underneath despite being opaque objects

**Comment by DeityLink** [comment]
Yeah that's how nanopaint works 

![286396360-84da06a5-4225-49cd-86ab-5236c2ad7d50](https://github.com/user-attachments/assets/f9a8353c-e126-4e2c-89ac-57b49f777fe2)

If you want lit up floor of various colors, without stuff on top being lit different colors then you gotta paint the floor first with white nanopaint, then paint on top with non-nanopaint "shapes". By using the half-tile shape you can cover a whole tile, without technically "replacing" the nanopaint, which lets you have tiles lit stuff on top of any specific color.

![image](https://github.com/user-attachments/assets/80d805b4-095d-4e52-ad62-cda12d7ebbca)

But yeah if you're just gonna use nano paint in stripes like that, stuff on top will be lit up in streaks, unless you shine a bright white light on top but that defeats the point of using nanopaint in the first place.

TL;DR: you're using nanopaint wrong

**Comment by Eneocho** [comment]
The issue is more that the colors blend wrong, the objects look semi-transparent, not lit.

**Comment by DeityLink** [comment]
That's an illusion, they're not actually transparent at all or you would see the full relief of the floor tiles through them. The only thing that shows up is the separation between the different stripes of nano paint, which again is the expected behaviour.

![image](https://github.com/user-attachments/assets/de99a5b1-7186-4c20-8d23-65163a7ac664)

**Comment by DeityLink** [comment]
It does look a bit weird on that flower but that's nothing that can't be explained by specific coats of nano paint applying different lights to it 
![image](https://github.com/user-attachments/assets/28118400-ae49-45d7-b58c-82725429b5bd)
